docs/resources/live_domain.md

```diff
@@ -0,0 +1,73 @@
+---
+subcategory: "Live"
+---
+
+# huaweicloud_live_domain
+
+Manages a Live domain within HuaweiCloud.
+
+## Example Usage
+
+### Create an ingest domain name and a streaming domain name
+
+```hcl
+variable "ingest_domain_name" {}
+variable "streaming_domain_name" {}
+
+resource "huaweicloud_live_domain" "ingestDomain" {
+  name = var.ingest_domain_name
+  type = "push"
+}
+
+resource "huaweicloud_live_domain" "streamingDomain" {
+  name               = var.streaming_domain_name
+  type               = "pull"
+  ingest_domain_name = huaweicloud_live_domain.ingestDomain.name
+}
+```
+
+## Argument Reference
+
+The following arguments are supported:
+
+* `region` - (Optional, String, ForceNew) Specifies the region in which to create the Live domain resource. If omitted,
+the provider-level region will be used. Changing this parameter will create a new resource.
+
+* `name` - (Required, String, ForceNew) Specifies the domain name. Changing this parameter will create a new resource.
```

Comment:
suggest to add more details about the domain name

Reply:
added